src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -97,15 +107,55 @@ SchemaProvider schemaProvider() {
     return new SchemaProvider(PredefinedSchemasGenerator.generate(schemaProperties()));
   }
 
-  private Properties schemaProperties() {
-    YamlPropertiesFactoryBean propertiesFactoryBean = new YamlPropertiesFactoryBean();
-    propertiesFactoryBean.setResources(new ClassPathResource("application.yml"));
-    propertiesFactoryBean.afterPropertiesSet();
-    return propertiesFactoryBean.getObject();
-  }
-
   @Bean
   MessageConverter messageConverter() {
     return new RecordMessageConverter();
   }
+
+  @Bean
+  @ConfigurationProperties(prefix = "sandbox.validation")
+  Schema schema() {
+    return new Schema();
+  }
+
+  @Value("${sandbox.transformation.xslt-url}")
```

Comment:
Here is the property where the url will come, but it has to be a valid url string, like 
`file:/Users/user/documents/path/to/xstl/default.xsl`